go.mod

```
@@ -1,6 +1,6 @@
 module github.com/grafana/mimir-proxies
 
-go 1.20
+go 1.19
```

```suggestion
go 1.20
```
Oh yeah - this'll need to go 🤣